### DISCUSSION TITLE
Vignette: pipeline with different data streams

### DISCUSSION THREAD

**Discussion by joshwlambert** [WHO-Collaboratory/collaboratory-epipipeline-community#5] [Ideas]
Identify tools, resources and potential blockers to facilitate activities on day 2 & 3.

**Comment by finlaycampbell** [comment]
https://github.com/epiverse-trace/episoap/blob/main/inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

**Comment by joshwlambert** [comment]
Useful Packages:
- [episoap](https://github.com/epiverse-trace/episoap)
- [godataR](https://github.com/WorldHealthOrganization/godataR)
- [sitrep](https://github.com/R4EPI/sitrep)

Useful Data:
- Data from godataR (.rds and .csv available in google drive here: https://drive.google.com/drive/u/0/folders/1qE2qflcZfFebGC8Ef_WtPJHP0t_N3xTr)
- Data from [global.health](https://global.health/)

Useful scripts:
- various R related scripts developed around Go.Data project which could be streamlined/packaged better:
https://worldhealthorganization.github.io/godata/R/
- in particular, flex dashboard example or daily sitrep example:  https://github.com/WorldHealthOrganization/godata-r-reports
-these need ALOT of work and are pretty messy.

**Comment by joshwlambert** [comment]
Plan:

- Select a few data streams
- Document a minimal standard for data 
- Get transmissibility pipeline working with multiple data streams
- Write vignette on how to use {episoap} with a novel data stream, useful for field epis to understand how to apply the pipelines
- Add severity pipeline to {episoap}
- Make severity pipeline work with multiple data streams
- Plan and add new pipelines to {episoap}

**Reply by joshwlambert** [reply]
Documentation on how to add a pipeline to the package.

**Comment by sarahollis** [comment]
some examples:
https://www.reconlearn.org/post/simulated-evd-early
https://www.reconlearn.org/post/practical-ebola-reconstruction.html

**Comment by joshwlambert** [comment]
[{episoap} R package](https://github.com/epiverse-trace/episoap)

**Comment by yuriei** [comment]
WHO mapping package (aligned with UN/WHO legal requirements for the international boundaries)
https://github.com/whocov/whomapper

**Comment by sarahollis** [comment]
here is the google doc we can add to: 
https://drive.google.com/drive/u/0/folders/1Rj7-1AvxeEeHqLWn-_oxnCTBasBTjZ5N

**Comment by CarmenTamayo** [comment]
Severity and ascertainment pipeline **https://github.com/CarmenTamayo/Applications-Epiverse-pipelines/blob/ctc-edits/Severity_ascertainment_case_studies.Rmd**

**Comment by joshwlambert** [comment]
https://docs.google.com/document/d/1A1_7cJtBMZx7LtUCWyPVCd3fwqvEnR8Jfin6tdYszzs/edit#heading=h.qrsuq5nunybt

https://github.com/joshwlambert/epipipe
https://joshwlambert.github.io/epipipe/
